specification/quantum/data-plane/Microsoft.Quantum/preview/2024-03-01-preview/quantum.json

```diff
@@ -199,7 +196,7 @@
           "200": {
             "description": "The request has succeeded.",
             "schema": {
-              "$ref": "#/definitions/JobDetails"
+              "$ref": "#/definitions/Quantum.Workspace.JobDetails"
```

Comment:
This is the reason we were not using "named" namespaces before for the common models (i.e. to avoid breaking changes in the name of the definitions).
But note that only the name of definitions is changing. Their actual schema is unchanged.